app/src/main/java/org/sopt/collaboration/campuspick/feature/home/HomeContract.kt

```diff
@@ -0,0 +1,13 @@
+package org.sopt.collaboration.campuspick.feature.home
+
+import org.sopt.collaboration.campuspick.core.ui.base.SideEffect
+import org.sopt.collaboration.campuspick.core.ui.base.UiState
+import org.sopt.collaboration.campuspick.domain.model.PopularActivity
+
+data class HomeState(
+    val popularActivity: List<PopularActivity> = emptyList(),
```

Comment:
```suggestion
    val popularActivity: List<PopularActivity> = immutableList(),
```
큰 차이는 없는데, 이렇게 써도 될 거 같아요